opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/aggregate.py

```diff
@@ -111,6 +122,11 @@ def merge(self, other):
             self.checkpoint = self._merge_checkpoint(
                 self.checkpoint, other.checkpoint
             )
+            if self.last_update_timestamp is None:
```

Comment:
If think this is/elif can also be condensed to a single if.

Reply:
I'm not sure we can do this since `other.last_update_timestamp` can be `None`